### PR TITLE
[ContentType] Fix parameters with encoded values

### PR DESCRIPTION
**Pull request by Maks3w**

### CODE REVIEW

File: src/Header/ContentType.php [hunk 1]
```diff
@@ -66,6 +74,13 @@ public function getFieldValue($format = HeaderInterface::FORMAT_RAW)
 
         $values = [$prepared];
         foreach ($this->parameters as $attribute => $value) {
+            if (HeaderInterface::FORMAT_ENCODED === $format &&
+                !Mime::isPrintable($value)) {
```
Comment (weierophinney) [inline src/Header/ContentType.php]: Push the `&&` to the second line, and the closing `)` and opening `{` onto a line by themselves.

File: src/Header/ContentType.php [hunk 2]
```diff
@@ -10,8 +10,9 @@
 namespace Zend\Mail\Header;
 
 use Zend\Mail\Headers;
+use Zend\Mime\Mime;
 
-class ContentType implements HeaderInterface
+class ContentType implements UnstructuredInterface
```
Comment (weierophinney) [inline src/Header/ContentType.php]: `UnstructuredInterface`? Unstructured… *what*, exactly?

**Comment by weierophinney** [comment]
Moved to 2.6.0, as it introduces a new interface.

File: src/Header/ContentType.php (hunk 2 above)
Comment (Maks3w) [inline src/Header/ContentType.php]: It's a requirement of HeaderWrap for to quote the header.

Comment (weierophinney) [inline src/Header/ContentType.php]: I missed that the interface already exists. Ugh. Okay, thanks; will change to the 2.5 release.

**Comment by Maks3w** [comment]
CS fix

**Comment by Maks3w** [comment]
I can ammend if you want

**Comment by weierophinney** [comment]
Accidentally merged only to develop, so cherry-picked the commits into master as 5ec3b92 a3bfd16 28e1399 and fb25989

**Pull request closed without merging**